paddle/fluid/operators/mkldnn/transpose_mkldnn_op.cc

```diff
@@ -223,8 +223,3 @@ REGISTER_OP_KERNEL(transpose_grad,
                    MKLDNN,
                    ::paddle::platform::CPUPlace,
                    ops::TransposeMKLDNNGradOpKernel<float>);
-
-REGISTER_OP_KERNEL(transpose2_grad,
```

Comment:
and we also need to remove the original impl of TransposeMKLDNNGradOpKernel here